osu.Game/Screens/Backgrounds/BackgroundScreenDefault.cs

```diff
@@ -102,6 +106,18 @@ private Background createBackground()
             else
                 newBackground = new Background(backgroundName);
 
+            switch (showSeasonalBackgrounds.Value)
+            {
+                case SeasonalBackgrounds.Sometimes:
+                    if (seasonalBackgroundLoader.IsInSeason())
+                        goto case SeasonalBackgrounds.Always;
```

Comment:
rather than using `goto`,  just write this as

`if (showSeasonalBackgrounds.Value == Always || (showSeasonalBackgrounds.Value == Sometimes))`